docs/agent/hostname-resolution.md

```diff
@@ -1,7 +1,11 @@
-_Edit 2018-31-08 : The default value of the new `hostname_fqdn` flag is planned to change in 6.6.0 instead of 6.4.0 to give users more time to take it into account._
+_Edit 2018-09-18 : Highlight the differences between Windows and other OSes._
+
+_Edit 2018-07-20 : The default value of the new `hostname_fqdn` flag is planned to change in 6.5.0 instead of 6.4.0 to give users more time to take it into account._
 
 # Difference in hostname resolution between Agent v5 and Agent v6 (<v6.6)
 
+## Linux
```

Comment:
I generalized a bit too much when I said `Linux` in my previous comment, this should really read `non-Windows platforms` 😅